config_template.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+OS_NAME=$(uname)
```

Comment:
Mhh I prefer keeping the OS_NAME in the build.sh, since it is not manually set anyway. Keep user settings in config.

Reply:
But not really important.

Reply:
Pro of having it in config:
- all the other building scripts have this variable automatically
Contra:
- indeed the config file might be intended only as a place for the variables that the user has to set... but not necessarily, in my opinion

In the end I think it's better having it in the config file, for laziness reasons!

Reply:
Good point.